services-manager/src/main/java/uk/gov/gchq/palisade/service/manager/service/ManagedService.java

```diff
@@ -36,6 +36,9 @@
     private final ManagedClient managedClient;
     private final Supplier<Collection<URI>> uriSupplier;
 
+    public static final int VALID_RESPONSE = 200;
+    public static final int ERROR_RESPONSE = 404;
```

Comment:
While it's out-of-scope, these should strictly come from the HttpStatus object

Reply:
quickly changed